docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -92,7 +92,25 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
       <td class="yes">Yes</td>
     </tr>
     <tr>
-      <td>Configure repository</td>
+      <td>Configure repository Git provider integration settings</td>
+      <td class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td class="yes">Yes</td>
+      <td class="maybe">Inherits original permission</td>
+      <td class="yes">Yes</td>
+    </tr>
+    <tr>
+      <td>Configure repository quality gates and goals</td>
+      <td class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td colspan="2" class="no">No</td>
+      <td class="yes">Yes</td>
+      <td class="maybe">Inherits original permission</td>
+      <td class="yes">Yes</td>
+    </tr>
+    <tr>
+      <td>Configure repository to run analysis on local build server, manage repository API tokens</td>
```

Comment:
Now that I'm reading it, I'm not convinced anymore 😅
We can either revert to your initial suggestion or reword this slightly. Up to you.

```suggestion
      <td>Set repository analysis to run on a local build server, manage repository API tokens</td>
```

Reply:
Using **Enable** instead, to match the description on the UI.

```suggestion
      <td>Enable repository analysis to run on a local build server, manage repository API tokens</td>
```